source/isaaclab/isaaclab/sensors/tacsl_sensor/visuotactile_sensor_cfg.py

```diff
@@ -0,0 +1,185 @@
+            sensor_cfg = VisuoTactileSensorCfg(render_cfg=GELSIGHT_R15_CFG)
+
+        Using custom sensor data::
+
+            custom_cfg = GelSightRenderCfg(
+                base_data_path="/path/to/my/sensors",
+                sensor_data_dir_name="my_custom_sensor",
+                image_height=480,
+                image_width=640,
+                mm_per_pixel=0.05,
+            )
+    """
+
+    base_data_path: str | None = None
+    """Base path to the directory containing sensor calibration data.
+
+    If ``None`` (default), downloads and caches data from Isaac Lab Nucleus directory.
+    If a custom path is provided, uses the data directly from that location without copying.
+    """
+
+    sensor_data_dir_name: str = "gelsight_r15_data"
+    """Directory name containing the sensor calibration and background data."""
+
+    background_path: str = "bg.jpg"
+    """Filename of the background image within the data directory."""
+
+    calib_path: str = "polycalib.npz"
+    """Filename of the polynomial calibration data within the data directory."""
+
+    real_background: str = "real_bg.npy"
+    """Filename of the real background data within the data directory."""
+
+    image_height: int = 320
+    """Height of the tactile image in pixels."""
+
+    image_width: int = 240
+    """Width of the tactile image in pixels."""
+
+    num_bins: int = 120
+    """Number of bins for gradient magnitude and direction quantization."""
+
+    mm_per_pixel: float = 0.0877
+    """Millimeters per pixel conversion factor for the tactile sensor."""
+
+
+##
+# Visuo-Tactile Sensor Configuration
+##
+
+
+@configclass
+class VisuoTactileSensorCfg(SensorBaseCfg):
+    """Configuration for the visuo-tactile sensor.
+
+    This sensor provides both camera-based tactile sensing and force field tactile sensing.
+    It can capture tactile RGB/depth images and compute penalty-based contact forces.
+    """
+
+    class_type: type = VisuoTactileSensor
+
+    # Sensor type and capabilities
+    render_cfg: GelSightRenderCfg = GelSightRenderCfg()
+    """Configuration for GelSight sensor rendering.
+
+    This defines the rendering parameters for converting depth maps to realistic tactile images.
+    Defaults to GelSight R1.5 parameters. Use predefined configs like GELSIGHT_R15_CFG or
+    GELSIGHT_MINI_CFG from isaaclab_assets.sensors for standard sensor models.
+    """
+
+    enable_camera_tactile: bool = True
+    """Whether to enable camera-based tactile sensing."""
+
+    enable_force_field: bool = True
+    """Whether to enable force field tactile sensing."""
+
+    # Force field configuration
+    num_tactile_rows: int = 20
+    """Number of rows of tactile points for force field sensing."""
+
+    num_tactile_cols: int = 25
+    """Number of columns of tactile points for force field sensing."""
+
+    tactile_margin: float = 0.003
+    """Margin for tactile point generation (in meters)."""
```

Comment:
Does this refer to the resolution of the tactile sensor? Margin makes it sound more of "margin" around the perimeter of the array.